website/content/en/preview/concepts/metrics.md

```diff
@@ -94,9 +94,15 @@ Pod state is the current state of pods. This metric can be used several ways as
 ### `karpenter_cloudprovider_duration_seconds`
 Duration of cloud provider method calls. Labeled by the controller, method name and provider.
 
-### `karpenter_cloudprovider_instance_price_estimate`
+### `karpenter_cloudprovider_instance_type_memory`
```

Comment:
```suggestion
### `karpenter_cloudprovider_instance_type_memory_bytes`
```

`kube_state` calls them `cpu_cores` and `memory_bytes`. I wonder if we should align with that: https://github.com/kubernetes/kube-state-metrics/blob/main/docs/pod-metrics.md

Reply:
It does this for `overhead` which is going to typically be much smaller numbers. Thinking in terms of `bytes` for an ec2 instance seems like the wrong scale -- I was leaning towards the other way and converting MiB to Gb, but passed on that.

I can change this if you feel strongly about it, but IMO reading really large numbers makes it difficult to understand this metric.

Reply:
I think for visualization solutions like Grafana, the assumption is that you will provide the raw value and then the visualization solution will be responsible for doing the conversion for you. That gives you the most flexibility with your data: https://community.grafana.com/t/grafana-unit-for-mb-gb-etc/68495/4

Reply:
Fair enough! I've changed this to `memory_bytes`.